### DISCUSSION TITLE
Mettre un clé Rtl sdr sur la applications rdio scanner

### DISCUSSION THREAD

**Discussion by gaspar1987** [chuot/rdio-scanner#210] [Q&A]
bonjour je veux avoir le rtl sdr sur la application  rdio scanner ?

**Comment by chuot** [comment]
So I'm not sure I understand your question, but if you're asking how to use an RTL dongle with Rdio Scanner, the answer is you can't. You are missing software that can record RF transmissions to a file, which Rdio Scanner can then process and distribute. These recorders are listed on the [main page](https://github.com/chuot/rdio-scanner) of the open source project.
